plugin/evm/evm_call_test.go

```diff
@@ -0,0 +1,91 @@
+package evm
+
```

Comment:
If these are integration tests and that they turn out to be slower/heavier than unit tests, we should make sure to add a build tag for it. This way when we have a CI for it, we could choose to not run the integration tests in some cases to save time.

```suggestion
package evm

//go:build integration
```